components/engagement/EngForm.vue

```diff
@@ -287,35 +319,56 @@ export default {
         { type: 'Scrum/Sprint' },
         { type: 'Advisory board/Council Meeting' }
       ],
-      showTag: false
+      showTag: false,
+      timeout: null,
+      attemptSubmit: false
+    }
+  },
+  validations: {
+    engagementDetail: {
+      subject: { required, maxChar: maxLength(50) },
+      type: { required },
+      date: { required },
+      description: { required },
```

Comment:
Description needs `maxlength(1000)` and a validation message.

Reply:
The text area is locked with maximum of 1000 characters. user cannot enter more than 1000 characters, but I can add the validation message for sure